ra/ra_test.go

```diff
@@ -2560,9 +2525,14 @@ func TestNewOrderExpiry(t *testing.T) {
 
 	// Create an order for that request
 	order, err := ra.NewOrder(ctx, orderReq)
+	fmt.Println(order)
```

Comment:
Looks like you have some debugging printfs left behind here.

Reply:
I do. I'm trying to fix this test still. I should have marked this as draft.